packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -320,17 +415,23 @@ const ActivityDetails = (props: Props) => {
           </div>
         )}
       </div>
-      {modalPortal(
+      {teamPickerModalPortal(
         <TeamPickerModal
           category={category}
           teamsRef={teams}
-          templatesRef={availableTemplates.edges
-            .map((edge) => edge.node)
-            .filter((template) => template.type === 'retrospective')}
-          closePortal={closePortal}
+          templatesRef={availableTemplates.edges.map((edge) => edge.node)}
+          closePortal={closeTeamPickerPortal}
           parentTemplateId={selectedTemplate.id}
+          type={type}
         />
       )}
+      {type === 'poker' &&
+        selectedTemplate.team.editingScaleId &&
+        pokerTemplateScaleDetailsPortal(
+          <div className='w-[520px]'>
```

Comment:
+1 do we have an issue for making these view mobile friendly? Anything over ~320px will probably take some extra consideration

Reply:
I'm not too worried about mobile here, since so few `new-meeting` views occur on mobile, and the existing mobile UX for editing templates like this is pretty bad.